packages/front-end/components/Experiment/EditHypothesisModal.tsx

```diff
@@ -0,0 +1,56 @@
+import { useForm } from "react-hook-form";
+import { useAuth } from "@/services/auth";
+import Modal from "../Modal";
+import Field from "../Forms/Field";
+
+interface Props {
+  source: string;
+  close: () => void;
+  experimentId: string;
+  hypothesis?: string;
```

Comment:
nit: should it be called `initialHypothesis` or `initialValue`?

Reply:
I think that's a good improvement. Thanks!